docs/learning/backend/Fonts.md

```diff
@@ -1,52 +1,86 @@
+A [FontFile]($backend) is a set of one or more font faces, usually originating in a file on disk. The file must be embedded into an iModel before any of the font families it contains can be used by text in that iModel.
 
-## FontType
+### Formats
 
-Most fonts in iModels are defined using the [OpenType](https://en.wikipedia.org/wiki/OpenType) format. However, for backwards compatibility with CAD applications, iModels also support the MicroStation "RSC" format, and the AutoCAD "SHX" format. The CAD format fonts are generally less capable, less complete, and less visually appealing. They should be avoided if possible, and are supported only for visual fidelity with legacy CAD data sources.
+iTwin.js supports three font file formats, represented by the [FontType]($common) `enum`:
+- [OpenType](https://en.wikipedia.org/wiki/OpenType), referred to throughout the API as "[TrueType](https://en.wikipedia.org/wiki/TrueType)", from which it derives. Virtually all modern fonts are distributed in this format. Common file extensions for OpenType fonts include .ttf, .otf, .ttc, and .otc, where the first two generally contain a single font face and the latter two a collection of multiple faces potentially belonging to multiple font families.
+- [SHX](https://help.autodesk.com/view/OARX/2024/ENU/?guid=GUID-DE941DB5-7044-433C-AA68-2A9AE98A5713), a primitive format originating in [AutoCAD](https://en.wikipedia.org/wiki/AutoCAD) and generally distributed as files with a .shx extension.
+- [RSC](https://docs.bentley.com/LiveContent/web/MicroStation%20Help-v27/en/GUID-FC78484C-E42F-30BF-BF68-2B2C025AE040.html), a primitive format originating in [MicroStation](https://en.wikipedia.org/wiki/MicroStation). In MicroStation, such fonts are defined in "resource" files with a .rsc extension.
 
-The enum [FontType]($common) specifies which of the 3 formats defines a font.
+SHX and RSC were designed for early [computer-aided design](https://en.wikipedia.org/wiki/Computer-aided_design) workflows and are hence sometimes referred to as "CAD fonts". They consist of a single font face generally comprising extremely simple representations of glyphs (e.g., basic line segments) and little to no support for Unicode characters. By contrast, OpenType fonts are generally more visually appealing, support Unicode, and provide multiple faces per family.
 
-### MicroStation RSC Fonts
+RSC fonts in iModels are stored in a binary format, not as .rsc files. No public API currently exists to convert to this representation, so for now the only way to embed a RSC font into an iModel is through a connector.
```

Comment:
I believe that "embed _a_ RSC font" should be "embed _an_ RSC font". I guess that "RSC" could be pronounced "resource", but I suspect that most people won't do that.